front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -1,19 +1,25 @@
+import { infraState, buildInfraStateReducers, InfraStateType } from 'reducers/infra';
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { OsrdStdcmConfState } from 'applications/operationalStudies/consts';
 import { Draft } from 'immer';
-import { defaultCommonConf, buildCommonConfReducers } from '../common';
+import { defaultCommonConf, buildCommonConfReducers } from '../osrdConfCommon';
 
-export const stdcmConfInitialState: OsrdStdcmConfState = {
+export type OsrdStdcmConfStateType = OsrdStdcmConfState & InfraStateType;
+
+export const stdcmConfInitialState: OsrdStdcmConfStateType = {
   maximumRunTime: 43200,
   standardStdcmAllowance: undefined,
+  stdcmMode: 'standard',
   ...defaultCommonConf,
+  ...infraState,
 };
 
 export const stdcmConfSlice = createSlice({
   name: 'stdcmConf',
-  initialState: stdcmConfInitialState,
+  initialState: { ...stdcmConfInitialState, ...infraState },
```

Comment:
Isn't `infraState` already in `stdcmConfInitialState` (line 14) ?
```suggestion
  initialState: stdcmConfInitialState,
```

Reply:
Done